samples/machine-learning/parallel-half-moons/README.md

```diff
@@ -15,7 +15,7 @@ The model is trained on a half-moons dataset, loaded in C# using the System.Text
 
 In this sample, the training loop is parallelized over model start points, with each model using its own instance of the full-state quantum simulator.
 Parallelizing in this way can lead to significantly improved performance, especially when using a large number of cores on a small number of qubits.
-The actual distribution of parallel tasks is performed using [Parallel LINQ (PLINQ)](https://docs.microsoft.com/dotnet/standard/parallel-programming/parallel-linq-plinq) from a C# host.
+The actual distribution of parallel tasks is performed using [Parallel LINQ (PLINQ)](https://docs.microsoft.com/en-us/dotnet/standard/parallel-programming/introduction-to-plinq) from a C# host.
```

Comment:
```suggestion
The actual distribution of parallel tasks is performed using [Parallel LINQ (PLINQ)](https://docs.microsoft.com/dotnet/standard/parallel-programming/introduction-to-plinq) from a C# host.
```
We should make sure to omit `en-us` so as to not override users' locale settings.